OneCommunityGlobal/HGNRest
# Shengwei_HotFix: Page render issue in Timelog

# Description
Hotfix blank page in the main site.

<img width="1093" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/69882989/0652a37e-2034-4c06-bcc4-c50caae4e710">


Hotfix for time entry render issue
- Add project/task name and category information in getTimeEntriesForSpecifiedPeriod
- Update time entry data model to include new virtuals

## Related PRS (if any):
Related to fronted [PR2159](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/2159)

## Main changes explained:
- Update timeEntryController.js and timeentry.js to include additional information for getTimeEntriesForSpecifiedPeriod API


## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. go to dashboard -> timelog
6. log a time entry under a specific project
7. remove the user from the project 
8. reload the timelog page

## Screenshots or videos of changes:

## Note:
Include the information the reviewers need to know.
